ui/apps/platform/src/Containers/Compliance/Dashboard/ComplianceDashboardPage.tsx

```diff
@@ -151,7 +151,7 @@ function ComplianceDashboardPage(): ReactElement {
                 </div>
             </PageHeader>
             <div className="flex-1 relative p-6 xxxl:p-8 bg-base-200" id="capture-dashboard">
-                {(inProgressScanDetected || error) && (
+                {(inProgressScanDetected || !!error) && (
```

Comment:
Many of the fixes in this PR are of the type where we conditionally render with `error && <></>`, which React 18 does not like since `error` tends to be an object.

In reality what we want is short circuiting when `error` is truthy, so I've made this be more explicit in these cases.